chore(markdown): Update option docs (#1812)
freesewing/freesewing: markdown/dev/reference/api/config/hide/en.md

title: hide
---

The `hide` key in the pattern configuration file allow you to configure

> parts the should be hidden by default.

Reviewer: ```suggestion
parts that should be hidden by default.
```

*Hidden* means that they will be drafted, but not rendered. This is

> ypically used for a base part on which other parts are built.

Reviewer: ```suggestion
typically used for a base part on which other parts are built.
```

Note that hidden parts will be rendered when the user requests 
to [only draft some parts of a pattern](/reference/api/settings/only) 
and includes the hidden part(s).

## Structure

> An array with strings that hold part names.

Reviewer: ```suggestion
An array of strings that holds part names.
```
